pkg/sql/colflow/colrpc/outbox.go

```diff
@@ -191,25 +201,27 @@ func (o *Outbox) Run(
 	}
 
 	log.VEvent(ctx, 2, "Outbox starting normal operation")
-	o.runWithStream(ctx, stream, cancelFn)
+	o.runWithStream(ctx, stream, flowCtxCancel, outboxCtxCancel)
 	log.VEvent(ctx, 2, "Outbox exiting")
 }
 
 // handleStreamErr is a utility method used to handle an error when calling
-// a method on a flowStreamClient. If err is an io.EOF, cancelFn is called. The
-// given error is logged with the associated opName.
+// a method on a flowStreamClient. If err is an io.EOF, outboxCtxCancel is
+// called, for all other error flowCtxCancel is. The given error is logged with
+// the associated opName.
 func (o *Outbox) handleStreamErr(
-	ctx context.Context, opName string, err error, cancelFn context.CancelFunc,
+	ctx context.Context, opName string, err error, flowCtxCancel, outboxCtxCancel context.CancelFunc,
 ) {
 	if err == io.EOF {
 		if log.V(1) {
-			log.Infof(ctx, "Outbox calling cancelFn after %s EOF", opName)
+			log.Infof(ctx, "Outbox calling outboxCtxCancel after %s EOF", opName)
 		}
-		cancelFn()
+		outboxCtxCancel()
 	} else {
 		if log.V(1) {
-			log.Warningf(ctx, "Outbox %s connection error: %+v", opName, err)
+			log.Warningf(ctx, "Outbox %s calling flowCtxCancel after connection error: %+v", opName, err)
```

Comment:
Let's remove the V(1) here - this is the critical error that we would have seen spammed to the logs too much on this event, which would have made this much easier to diagnose.